lib/OpenQA/YAML.pm

```diff
@@ -91,8 +91,13 @@ sub validate_data {
         }
     }
     catch {
-        # The first line of the backtrace gives us the error message we want
-        push @errors, (split /\n/, $_)[0];
+        if (m/^YAML::XS::Load Error/) {
+            push @errors, $_;
+        }
+        else {
+            # The first line of the backtrace gives us the error message we want
+            push @errors, (split /\n/, $_)[0];
```

Comment:
I suppose [using regex](https://github.com/os-autoinst/openQA/pull/3096/files#diff-2559347dfdfd46e3d678b4f8823f2d7fR577) would be faster.

Reply:
Yeah, probably.
I didn't introduce this line, and it's only happening when something is wrong with the schema, so it should only happen in development anyway.

Reply:
I added a LIMIT to the split. Ok?